src/Shared/Recruit.Vacancies.Client/Application/Services/INotifyVacancyUpdates.cs

```diff
@@ -6,5 +6,6 @@ namespace Esfa.Recruit.Vacancies.Client.Application.Services
     public interface INotifyVacancyUpdates
     {
         Task VacancyManuallyClosed(Vacancy vacancy);
+        Task VacancyExtended(Vacancy vacancy);
```

Comment:
If the vacancy's start date has been changed then it hasn't really been extended.  Should we just call this `LiveVacancyChanged`

Reply:
👍